src/main/java/org/jenkinsci/plugins/mesos/MesosApi.java

```diff
@@ -287,6 +291,11 @@ private void updateState(StateEventOrSnapshot event) {
     }
   }
 
+  /** test method to set the agent timeout duration */
```

Comment:
What do you mean by "test"?

Reply:
ah just mean to be used for test purposes